Library/Formula/mpd.rb

```diff
@@ -116,4 +119,11 @@ def plist; <<-EOS.undent
     </plist>
     EOS
   end
+
+  test do
+    pid = fork { exec "#{bin}/mpd --stdout --no-daemon --no-config" }
+    sleep 2
+    Process.kill("SIGINT", pid)
+    Process.wait(pid)
+  end
```

Comment:
Garg. Test succeeds even when it shouldn't. Will have to try and find another way, if there is one.